sdk/js/src/Client.ts

```diff
@@ -207,6 +206,7 @@ export class DevCycleClient<
                     Date.now(),
                 )
             }
+            this._isInitialized = true
```

Comment:
this seems to be doing something different than the title of the PR. Are we trying to make this false when the SDK key is invalid, or when the config request fails for any reason?

Reply:
ah, that's right.. this should only be when the config-key is invalid.. will update